pallets/account-manager/src/manager.rs

```diff
@@ -100,6 +196,74 @@ impl<T: Config> AccountManagerExt<T::AccountId, BalanceOf<T>> for Pallet<T> {
 
         Ok(())
     }
+
+    /// Collect claimable (only SFX execution rewards) for Executors and Stakers submitted by Circuit at the duration of the current Round
+    fn on_collect_claimable(
```

Comment:
i think this should just take all ledger items and claim them when the storage items are better generalised. really this pallet shouldn't care about who is who, I think.

Or probably this should have a todo to move the executor specific logic to pallet-executors and remove the storage items. but really that will be more hassle with storage migrations than to just add a pallet and remove those coupled items.

Reply:
agreed, will expose another method `fn return_claims()` from AccountManager